ci/test.sh

```diff
@@ -102,5 +102,13 @@ convox ps stop $ps -a httpd
 convox ps -a httpd | grep -v $ps
 convox deploy -a httpd
 
+# timers
+sleep 30
+
+timerLog=$(convox logs -a httpd --no-follow --since 1m | grep service/web/timer-example)
+if ! [[ $timerLog == *"Hello Timer"* ]]; then
+  echo "failed"; exit 1;
+fi
+
```

Comment:
Use the same pattern as in the other tests.

E.g.:
```suggestion
convox logs -a httpd --no-follow --since 1m | grep service/web/timer-example | grep ""Hello Timer"
```